src/arns.lua

```diff
@@ -925,7 +940,11 @@ function arns.recordIsActive(record, currentTimestamp)
 		return true
 	end
 
-	return record.endTimestamp and record.endTimestamp >= currentTimestamp
+	-- record starts before the current timestamp and ends after the current timestamp
+	return record.startTimestamp
+		and record.startTimestamp <= currentTimestamp
```

Comment:
Nit: maybe update currentTimestamp to just timestamp or atTimestamp since this is now a general purpose function that can look into the past.

Reply:
[dfad08c](https://github.com/ar-io/ar-io-network-process/pull/352/commits/dfad08cb6ad883250bfd28115cd0220df81748de)